src/Behat/Behat/Transformation/Transformer/RepositoryArgumentTransformer.php

```diff
@@ -160,7 +160,7 @@ private function splitSimpleAndNormalTransformations(array $transformations)
         return array_reduce($transformations, function ($acc, $t) {
             return [
                 $t instanceof SimpleArgumentTransformation ? array_merge($acc[0], [$t]) : $acc[0],
-                !$t instanceof SimpleArgumentTransformation ? array_merge($acc[1], [$t]) : $acc[1],
+                $t instanceof SimpleArgumentTransformation ? $acc[1] : array_merge($acc[1], [$t]),
             ];
         }, [[], []]);
```

Comment:
This existing logic took a bit of figuring out! It would probably be easier to follow as a pair of simple array_filters (and if the array_reduce was originally a performance concern I'm not sure the repeated array_merge makes it any better).

But I agree this change is correct for the existing implementation.